backend/db/init_db.py

```diff
@@ -15,6 +16,9 @@ def _init_postgres_db(app):
 
     db.session.commit()
 
+    # Seed database with sample data
+    seed()
```

Comment:
Leave a comment here to take this out once we have real data. This should be in a setup doc, but we'll leave it for now.